src/torchmetrics/text/infolm.py

```diff
@@ -38,6 +38,21 @@
     __doctest_skip__ = ["InfoLM", "InfoLM.plot"]
 
 
+_Information_Measure_Higher_Is_Better = {
```

Comment:
can we rather have somehow dynamic, with import the class and ask for its attribute?
my point is not to duplicate this and also prevent forgetting update this part when new metric is added...

Reply:
something like:
```py
def _information_measure_higher_is_better(name):
    cls = impotlib.load(f'torchmetrics.retrival.{name}')
    return getatr(cls, 'higher_is_better')
```

Reply:
not sure I understand why you would want it like that?  

was thinking it might make slightly more sense to have it be part of something in `functional/text/infolm.py` but didn't see anything similar so figured I should make the PR/diff as small as possible since it didnt seem to apply to other metrics

Reply:
my main point it so load it dynamically without creating another hard-ded Look-up-table

Reply:
still not sure I follow.  the value will be hardcoded just in another file? and why `torchmetrics.retrival.{name}` instead of off of something in `src/torchmetrics/functional/text/infolm.py`?

Reply:
The `higher_is_better` is already an attribute for each class, right?
so setting here again `false/true` is duplicate...

Reply:
@borda I think maybe you are missing the point here that the attribute depends on the input arguments in the class (usually our metrics do not). So I do think the implementation makes sense here.

Reply:
My point is not to map to bools but if needed to class names and source the bools directly...

Reply:
After discussion with @Borda offline we have come to the conclusion that the current implementation was fine (except that I have moved the dict to be a private attribute of the metric).